pkg/virt-config/featuregate/active.go

```diff
@@ -0,0 +1,123 @@
+	AlignCPUsGate = "AlignCPUs"
+
+	// VolumesUpdateStrategy enables to specify the strategy on the volume updates.
+	VolumesUpdateStrategy = "VolumesUpdateStrategy"
+	// VolumeMigration enables to migrate the storage. It depends on the VolumesUpdateStrategy feature.
+	VolumeMigration = "VolumeMigration"
+	// Owner: @xpivarc
+	// Alpha: v1.3.0
+	//
+	// NodeRestriction enables Kubelet's like NodeRestriction but for Kubevirt's virt-handler.
+	// This feature requires following Kubernetes feature gate "ServiceAccountTokenPodNodeInfo". The feature gate is available
+	// in Kubernetes 1.30 as Beta.
+	NodeRestrictionGate = "NodeRestriction"
+)
+
+func init() {
+	RegisterFeatureGate(FeatureGate{Name: ExpandDisksGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: CPUManager, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: NUMAFeatureGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: IgnitionGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HypervStrictCheckGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: SidecarGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: GPUGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HostDevicesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: SnapshotGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMExportGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HotplugVolumesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HostDiskGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VirtIOFSGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DownwardMetricsFeatureGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: Root, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: ClusterProfiler, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: WorkloadEncryptionSEV, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DockerSELinuxMCSWorkaround, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VSOCKGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DisableCustomSELinuxPolicy, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: KubevirtSeccompProfile, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DisableMediatedDevicesHandling, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HotplugNetworkIfacesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: PersistentReservation, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMPersistentState, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: MultiArchitecture, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMLiveUpdateFeaturesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: BochsDisplayForEFIGuests, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: NetworkBindingPlugingsGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: AutoResourceLimitsGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: CommonInstancetypesDeploymentGate, State: Alpha})
```

Comment:
This should be Beta and is about to be GA'd through https://github.com/kubevirt/kubevirt/pull/12753 if that lands ahead of this PR.